js/main.js

```diff
@@ -25,9 +25,86 @@ function getRandomFractionNumber(min, max, numberSymbols) {
 
   const randomNumber = min + Math.random() * (max - min);
 
-  return Number(randomNumber.toFixed(1 * numberSymbols));
+  return Number(randomNumber.toFixed(numberSymbols));
 }
 
-getRandomFractionNumber(1, 10, 2);
+getRandomFractionNumber(1, 10, 0);
 
 
+// Возвращает фотографию пользователя
+const AVATAR_COUNTS = 10;
+
+function getAvatarUrl () {
+  const randomNumber = getRandomFractionNumber(1, AVATAR_COUNTS, 0);
+
+  let AvatarUrl = 'img/avatars/userget' + `0${ randomNumber }.png`;
+  if (randomNumber === 10) {
+    AvatarUrl = 'img/avatars/userget' + `${ randomNumber }.png`;
+  }
+
+  return AvatarUrl;
+}
+
+const avatarArray = [];
+
+function getavatarValue () {
+  let nextValue;
+  while (avatarArray.length < AVATAR_COUNTS) {
+    nextValue = getAvatarUrl();
+    if(!avatarArray.includes(nextValue))
+    {avatarArray.push(nextValue);
+      return nextValue;
+    }
+  }
+}
+
+const createAvatarArray = Array.from({length: 10}, getavatarValue);
+
+
+// возвращает время заезда и выезда
+function getCheckinAndCheckout () {
+  const randomNumber = getRandomFractionNumber(12, 14, 0);
+  const checkinAndCheckout = `${ randomNumber }` + ':00';
+  return checkinAndCheckout;
+}
+
+// Массивы с рандомными данными
+const title = ['Квартира в центре Питера', 'Квартира в центре Москвы', 'Квартира в центре Сочи'];
```

Comment:
Именование всех констант (неизменяемых внутри, то что мы условно определяем за вещи, которые даже если и может, то не должны менять) нужно назвать по критериям в верхнем регистре и перенести в верх кода, плюс по критериям нужно во множественном числе называть, это массив значений, т.е. `TITLES` в идеале ещё и связать с типом, например `OFFER_TITLES`
```suggestion
const OFFER_TITLES= ['Квартира в центре Питера', 'Квартира в центре Москвы', 'Квартира в центре Сочи'];
```